tests/benchmark/plot.py

```diff
@@ -0,0 +1,24 @@
+import matplotlib.pyplot as plt
+import subprocess
+import sys
+
+ret = []
+for prog in sys.argv[1:]:
+	ret.append([[], []])
+	for _ in range(10): # run 10 times
+		s = subprocess.run(['/usr/bin/time', '--format', "%e %M", 'enarx', 'deploy', prog], capture_output=True)
```

Comment:
Can't we use `python` directly to do this instead of `/usr/bin/time`?
Looks like `psutil` provides exactly what we need in a cross-platform fashion https://psutil.readthedocs.io/en/latest/

Otherwise, if you don't want to do that, I'm fine with replacing `/usr/bin/time` with a direct `time` call for now and `psutil` migration in a follow-up (which does not need to be completed by you)
Coming back to https://github.com/enarx/codex/pull/75#discussion_r1041352620, since this is not running from within a shell, using `time` directly should be fine, otherwise `/usr/bin/env time` is another option, which is guaranteed to execute the `time` binary. Direct `time` call is preferable if it works

Reply:
I'm not sure how to implement this in python because i use time to also get memory usage

Reply:
No problem, let's just use `time` directly then and the rest looks good

Reply:
done!